notebooks/05_operationalize/pipeline-demo/aml-pipeline-use-datebricks-as-compute-target.ipynb

```diff
@@ -0,0 +1,697 @@
+{
```

Comment:
<p>throughout this notebook, we should retrieve config from env vars without defaults, this will force them to be set up properly</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/643/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>